.github/workflows/docs.yml

```diff
@@ -125,4 +129,9 @@ jobs:
         run: |
           cmake -B lld-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="lld" -DLLVM_ENABLE_SPHINX=ON ./llvm
           TZ=UTC ninja -C lld-build docs-lld-html
+      - name: Build OpenMP docs
+        if: steps.docs-changed-subprojects.outputs.openmp_any_changed == 'true'
+        run: |
+          cmake -B openmp-build -GNinja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS="clang;openmp" -DLLVM_ENABLE_SPHINX=ON ./llvm
```

Comment:
Why do we need to enable clang here?

Reply:
The `CMake` invocation fails if you don't also build `clang`. I'm not entirely sure why the dependency tree is set up the way it is, but it's needed to get the build working.

Reply:
Did you try `-DENABLE_RUNTIMES=openmp` ?

Reply:
Yes. I tried doing a runtimes build similar to `libc++`/`libc`/`libunwind`, but didn't have any luck with that (didn't get past CMake configuration). The buildbot for publishing the OpenMP docs also builds with `openmp` as a project instead of as a runtime, and I suspect that's for a reason.